src/main/java/uk/gov/hmcts/reform/em/stitching/batch/DocumentTaskCallbackProcessor.java

```diff
@@ -50,33 +54,51 @@ public DocumentTaskCallbackProcessor(OkHttpClient okHttpClient, AuthTokenGenerat
 
     @Override
     public DocumentTask process(DocumentTask documentTask) throws InterruptedException {
+
+        int retryCount = 1;
+        Response response = null;
+
         try {
 
-            Thread.sleep(callBackDelayMilliseconds);
-            Request request = new Request.Builder()
+            while (retryCount <= callBackMaxAttempts) {
```

Comment:
I'm a bit confused, why is this a while loop? I thought the task gets picked up, we try to execute, if that fails and we haven't reached the max no of retries we just don't update it so that it can get picked up again